content/docs/command-reference/update.md

```diff
@@ -7,8 +7,9 @@ import `.dvc` files.
 ## Synopsis
 
 ```usage
-usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R] [--no-download] [--to-remote]
-                  [-r <name>] [-j <number>] targets [targets ...]
+usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R] [--no-download]
+                  [--to-remote] [-r <name>] [-j <number>]
+                  targets [targets ...]
```

Comment:
Suggested reorg of `update` options cc @dberenbaum WDYT?

```suggestion
usage: dvc update [-h] [-q | -v] [-R] [--rev <commit>]
                  [--to-remote] [-r <name>]
                  [--no-download] [-j <number>]
                  targets [targets ...]
```

Reply:
Looks good to me. Waiting to reach an agreement on this and then will open a PR on the dvc side

Reply:
@dtrifiro Can `--to-remote` and `--no-download` both be used? What happens? Should they be exclusive?

Reply:
They are exclusive, just like `--no-exec` (see [here](https://github.com/iterative/dvc/blob/d762372d555536ef7c89df10aa5dbaa3dd670e62/dvc/repo/imp_url.py#L39-L47)

Reply:
Thanks! No strong opinion here from me.

Reply:
Updated in the docs. I think we don't need to take any action on the dvc side.

Reply:
> They are exclusive

Q: Shouldn't the help output print them like `[--to-remote | --no-download]` ?

Reply:
We could do that, although it's not a syntax we widely use in other dvc commands' help. The only place we currently use it (apart from quet/verbose flags, `-q | -v`) is `import-url [--no-exec | --no-download]`. I have no strong preference, but I think it's fine like this.

Reply:
@jorgeorpinel I had the same thought initially, but then when I looked closer, I saw that in `import-url`, `import`, and `update`, we keep `[--no-exec | --no-download]` together where applicable but keep `--to-remote` separate since it is paired with the `-r` argument. I think it's fine as is.